src/braket/default_simulator/noise_operations.py

```diff
@@ -0,0 +1,290 @@
+from __future__ import annotations
+
+from typing import Tuple
+
+import braket.ir.jaqcd as braket_instruction
+import numpy as np
+
+from braket.default_simulator.operation import KrausOperation
+from braket.default_simulator.operation_helpers import (
+    _from_braket_instruction,
+    check_cptp,
+    check_matrix_dimensions,
+    ir_matrix_to_ndarray,
+)
+
+
+class BitFlip(KrausOperation):
+    """Bit Flip noise channel"""
+
+    def __init__(self, targets, probability):
+        self._targets = tuple(targets)
+        self._probability = probability
+
+    @property
+    def matrices(self) -> np.ndarray:
+        k0 = np.sqrt(1 - self._probability) * np.array([[1, 0], [0, 1]])
+        k1 = np.sqrt(self._probability) * np.array([[0, 1], [1, 0]])
+        return [k0, k1]
```

Comment:
Define these in `__init__` as `self._matrices` and just return `self._matrices` in this method. Same goes for all other `KrausOperation`s.